tox.ini

```diff
@@ -1,44 +1,77 @@
+# This is configuration file of tox, which run unit tests and other checks.
+# See: https://tox.readthedocs.io/en/latest/index.html
+
 [tox]
 minversion = 2.0
 envlist = py34,py27,pypy,pep8
-skipsdist = True
 
+# Test env defaults, runs unit tests via pytest.
+# In this case, the "default" means that py34, py27 or other test enviroment
+# without explicit definition would use this configuration.
 [testenv]
-usedevelop = True
-install_command = pip install -U {opts} {packages}
-setenv = VIRTUAL_ENV={envdir}
-deps = -r{toxinidir}/requirements.txt
-       -r{toxinidir}/test-requirements.txt
-       git+https://github.com/Tendrl/common.git
-commands =
-    apt-get install build-essential \
-        libsystemd-journal-dev \
-        libsystemd-daemon-dev \
-        libsystemd-dev
+# usedevelop = true
+# Uncomment the previous line when you need to speedup unit tests setup
+# during development, it would disable sdist build (it sets skipsdist = true)
+# and install the project in devel mode (pip install -e .).
+deps =
+    pytest
+    mock
+commands = py.test tendrl
 
+# Runs PEP8 checks on the source code via flake8 tool
 [testenv:pep8]
-commands = flake8 {posargs}
-
-[testenv:venv]
-commands = {posargs}
+skip_install = true
+deps = flake8
+commands = flake8 tendrl
 
+# Runs py.test unit tests with coveralls.io cov-report
+# used by Travis CI, see .travis.yml)
 [testenv:cover]
 passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps =
+    pytest
+    pytest-cover
+    coverage
+    coveralls
 commands =
     py.test --cov-report= --cov=tendrl tendrl/alerting/tests
     coveralls --verbose
```

Comment:
Coveralls wasnt giving coverage per branch, hence we are trying CodeCov, please use that instead of coveralls

https://github.com/Tendrl/node-agent/blob/develop/tox.ini#L30